currency-exchange-service/readme.md

```diff
@@ -1,28 +1,24 @@
+About
+=====
+- Implement `/from/{from}/to/{to}` end point which get two countries name and return exchange rate between them
+
 Dependencies
 ============
-
 - #### Web (embedded tomcat + MVC)
     - spring-boot-starter-web
-- #### Dev Tools
-    - spring-boot-devtools
-    - lombok
-    - spring-boot-configuration-processor
-- #### Dev Ops
-    - spring-boot-starter-actuator
-- #### Data
-    - spring-boot-starter-data-jpa
-    - mysql-connector-java
 - #### Cloud
-    - ##### spring-cloud-starter-config
-    - ##### spring-cloud-starter-netflix-eureka-client
-    - ##### spring-cloud-starter-zipkin
+  - ##### spring-cloud-starter-config
```

Comment:
replace spring-cloud-starter-config like #### cloud config and add dependecny below it